velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -155,11 +154,11 @@ std::shared_ptr<const HiveWriterParameters> HiveDataSink::getWriterParameters(
       break;
     }
     default:
-      VELOX_UNREACHABLE();
+      VELOX_UNREACHABLE(commitStrategyToString(commitStrategy_));
```

Comment:
Guess no need for an explicit value at least with VELOX_UNREACHABLE(). Sounds contradicting.

Reply:
I think it is helpful to log out the actual value in this case.